devops/sql/creditRisk.sql

```diff
@@ -0,0 +1,390 @@
+
```

Comment:
Same q as above, service name and documentation says "credit-application", not "credit risk"